scenario/scenario_ontology.ttl

```diff
@@ -0,0 +1,23 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@Prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
```

Comment:
prefix should be lowercase

Reply:
Fixed in [6a622d4](https://github.com/GAIA-X4PLC-AAD/ontology-management-base/commit/6a622d4fdd24480f16758d3943524710e5b06740)